src/variants.php

```diff
@@ -69,36 +69,46 @@
 
 
 
-if (!ACTION && (empty($_PE[1]) || preg_match('/^chr[0-9A-Z]{1,2}$/', $_PE[1]))) {
+if (!ACTION && (empty($_PE[1]) ||
+    preg_match('/^(chr[0-9A-Z]{1,2})(:([0-9,]+)-([0-9,]+))?$/', $_PE[1], $aRegionArgs))) {
```

Comment:
Do we really want to support the `,` in the positions? It's something ugly the genome browsers are doing, but none of our APIs support it. Was this requested, or did it just seem useful? If we support it here, should our APIs also support it?

Oh, and could you add one more tab to this line, so it's clear we're still in an `if` that is not on the same level as `!ACTION`?